CHANGELOG.md

```diff
@@ -69,6 +69,12 @@ This project adheres to [Semantic Versioning](http://semver.org/).
   optional arguments.
   (#[1242](https://github.com/nix-rust/nix/pull/1242))
 
+- Removed `unistd::daemon`, `unistd::pipe2`,
+  `sys::event::FilterFlag::NOTE_EXIT_REPARENTED`, and
+  `sys::event::FilterFlag::NOTE_REAP` on OSX and ios.  Removed
```

Comment:
I'd split these as separate line items since they aren't really related even though they're part of the same PR.